### DISCUSSION TITLE
Attacks that cause bleeding (ex. Feral Predator) and poison (ex. Vivisector) ignore armour

### DISCUSSION THREAD

**Discussion by a2ZxgdqHeSs8F51cIyuK** [cataclysmbn/Cataclysm-BN#2075] [General]
### Is your feature request related to a problem? Please describe.

The enemy types in the title can cause bleeding and poison without taking armour into account, a character will bleed or be poisoned regardless of if they have no armour or if they are so heavily armoured that they are next to impervious to most damage

### Describe the solution you'd like

As described by "Kheir#9791" on Discord:
"Bleeding is an effect, so we'd need to rewire bleed.
**We can make effects only trigger if you actually take damage from the attack** (with flag allowances for non-damaging effects like gasses or similar), but that would still make bleed all or nothing even if you only take 1 point of damage."
If enemies had to actually "defeat" the player's armour, even by only one point, that would already go a long way in improving the experience related to these types of enemy.

### Describe alternatives you've considered

A more complex rework of the bleeding system has been suggested as a long term goal.

**Comment by Coolthulhu** [comment]
This is partly intentional and partly overpowered.

Those enemies just have great armor piercing capabilities, so their effects get through.
Predators have percentage armor piercing that may be too high, vivisectors have acid damage, which isn't handled well right now due to acid resistance being calculated off the ugly environment protection value.
